src/vnsw/agent/port_ipc/port_ipc_handler.cc

```diff
@@ -498,12 +501,16 @@ bool PortIpcHandler::GetPortInfo(const string &uuid_str, string &info) const {
 }
 
 bool PortIpcHandler::InterfaceExists(const std::string &name) const {
-#if 0 //WINDOWS-TEMP
-    int indx  = osspecific_if_nametoindex(name.c_str());
+
+#ifdef _WINDOWS
+    int indx  = windows_if_nametoindex(name.c_str());
```

Comment:
Agreed. But intentionally created a wrapper in case the type of NET_IFINDEX  is not unsigned int as in linux